src/replica/duplication/load_from_private_log.cpp

```diff
@@ -157,6 +159,7 @@ void load_from_private_log::find_log_file_to_start()
         error_s es = log_utils::open_read(pr.second->path(), file);
         if (!es.is_ok()) {
             LOG_ERROR_PREFIX("{}", es);
+            _duplicator->set_duplication_plog_checking(false);
```

Comment:
There's `return` statement in function `find_log_file_to_start(log_file_map)`, which should be also set false:

```c++
void load_from_private_log::find_log_file_to_start(
    const mutation_log::log_file_map_by_index &log_file_map)
{
    _current = nullptr;
    if (dsn_unlikely(log_file_map.empty())) {
        LOG_ERROR_PREFIX("unable to start duplication since no log file is available");
        return;
    }
```
```